extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/handler/FSInternetRadioHandler.java

```diff
@@ -114,7 +116,7 @@ public void run() {
         }
     };
 
-    public FSInternetRadioHandler(Thing thing) {
+    public FSInternetRadioHandler(Thing thing, CommonHttpClient httpClient) {
```

Comment:
Do you miss to set the assignment?

```
this.httpClient = httpClient;
```

Fields that are set by the constructor and immutable should be marked as final.
You would get an error that it is not set.